code-of-conduct.md

```diff
@@ -0,0 +1,73 @@
+
+Topluluklar, içinde bulundukları toplumları yansıtırlar ve toplumda var olan birçok eşitsizlik ve güç ihlallerine karşı koymak için olumlu eylemler şarttır.
+
+Topluluğumuzda sıcak, samimi ve tüm katılımcıları tam anlamıyla katkıda bulunmaya teşvik etmek için ekstra çaba harcayan birini görürseniz, bilmek isteriz.
+
+
+Beklenen Davranışlar
+-----------------
+
+* Aktif bir katılımcı olun. Bunu yaparak, bu topluluğun sağlığına ve ömrüne katkıda bulunmuş olursunuz.
+* Konuşma ve eylemlerinizde dikkatli ve saygılı olmaya özen gösterin.
+* Fikir ayrılıklarında işbirlikçi olun.
+* Aşağılayıcı, ayrımcı veya taciz edici davranış ve konuşmalardan kaçının.
+* Çevrenize ve diğer katılımcılara dikkat edin. Tehlikeli bir durum, sıkıntılı bir kişi veya bu Davranış Kurallarının ihlal edildiğini görürseniz, önemsiz olsa bile lütfen topluluk liderlerini uyarın.
+
+
+Kabul Edilemez Davranışlar
+---------------------
+
+Bu davranışlar kabul edilemez: topluluğumuzdaki herhangi bir katılımcının çevrimiçi olarak, ilgili tüm etkinliklerde ve topluluk işi bağlamında yapılan bire bir görüşmelerde korkutucu, taciz edici, küfürlü, ayrımcı, aşağılayıcı veya küçümseyici konuşma veya eylemler. Etkinliğin gerçekleştiği mekan halka açık olabilir; lütfen bu mekanlardaki tüm insanlara saygılı olmaya özen gösterin.
+
+Bu davranışlar taciz olarak kabul edilir: cinsiyet, cinsel yönelim, ırk, din, engellilik durumu ile ilgili aşağılayıcı veya önyargılı sözlü veya yazılı yorumlar; çıplaklık ve/veya cinsel imgelerin uygunsuz kullanımı (sunum slaytları dahil); uygunsuz şiddet tasvirleri (sunum slaytları dahil); gözdağı vermek, gözetlemek veya takip etmek; taciz edici fotoğraf veya video; konuşmaların veya diğer olayların kasıtlı olarak bozulması; uygunsuz fiziksel temas ve uygunsuz cinsel ilgi.
+
+
+Kabul Edilemez Davranışların Sonuçları
+-------------------------------------
+
+Sponsorlar ve yetkili/görevli olanlar da dahil olmak üzere herhangi bir topluluk üyesinin kabul edilemez davranışta bulunmasına müsamaha edilmeyecektir.
+Uyarılan kişinin bu davranışı derhal durdurması ve uyum sağlaması beklenir.
+
+Topluluk üyelerinden birisi kabul edilemez davranışlarda bulunursa, topluluk organizatörleri, uyarı vermeden (ve ücretli bir etkinlik durumunda geri ödeme yapmadan) topluluktan geçici olarak uzaklaştırma veya kalıcı olarak çıkarma dahil olmak üzere, uygun gördükleri herhangi bir eylemde bulunabilirler.
+
+
+Kabul Edilemez Davranışlara Tanık Olursanız veya Maruz Kalırsanız
+------------------------------------------------------
+
+Kabul edilemez davranışa maruz kalırsanız veya tanık olursanız, veya başka endişeleriniz varsa, lütfen en kısa sürede bir topluluk organizatörüne ulaşın.
+İletişim kurabileceğiniz organizatörlerin listesini sayfanın alt tarafında görebilirsiniz. Bunlara ek olarak, organizatörler, tacize uğrayan/kabul edilemez davranışa tanık olan/maruz kalan topluluk üyelerinin kendilerini güvende hissetmelerine yardımcı olacaklardır.
```

Comment:
Burada bahsi geçen liste kısmını göremedim ben sanırım o gözden kaçmış

Reply:
Burda markdown dosyasinin en altina seni ekleyelim mi abi @enderahmetyurt ? Yoksa core team'den bir kac kisi mi yer alsin?

Reply:
neden birilerini ekliyoruz anlamadim

Reply:
İletişim için kişi yerine bir mail adresi verebiliriz bence, mail adresi bir grup olur core team'e yönlendirir diye düşünüyorum. Kişilere kontakt vermek hoş olur @enderahmetyurt

Reply:
Bence de herhangi bir kişi ismi geçmesine gerek yok.

Reply:
Kişi bilgisi yerine bir kontakt noktası vermeye ne dersiniz yani bir email adresi gibi? Bu gerekli bence sizce? @tayfunoziserikan @enderahmetyurt

Reply:
bence email yeterli. normalde rubyturkiyeorg@gmail.com kullaniyoruz.